user/api.go

```diff
@@ -814,7 +814,7 @@ func (a *Api) ServerLogin(res http.ResponseWriter, req *http.Request) {
 // @Failure 401 {string} string ""
 // @Router /login [get]
 func (a *Api) RefreshSession(res http.ResponseWriter, req *http.Request) {
-
+	a.logger.Printf("refresh session token %v", req)
```

Comment:
This log is good for debug & prod.
But maybe the request content leak too much in the log?

Reply:
yep agree, i removed it and add trace token as discussed 😉